### PR TITLE
docs: release notes for v0.1.0

### PR DESCRIPTION
**Pull request by jecluis**
Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

File: docs/release-notes/s3gw-v0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Should a bug be found and not expected to be related to the list below, one
+should feel encouraged to file an issue in our
+[github repository](https://github.com/aquarist-labs/s3gw-core).
+
+
+## What we support
+
+At the moment we support creating buckets, doing basic operations on objects
+(PUT/GET/DELETE), and listing bucket contents.
```
Comment (0xavi0) [inline docs/release-notes/s3gw-v0.1.0.md:16]: We also support recursive DELETE (although the bucket itself is not deleted).
I'm not sure if it worths mentioning it (just to extend this section a bit more)

File: docs/release-notes/s3gw-v0.1.0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Release Notes - v0.1.0
+
+This is the first publicly available iteration of the s3gw project. Bugs are to
+be expected (and welcomed!), and performance to be bad at best.
```
Comment (giubacc) [inline docs/release-notes/s3gw-v0.1.0.md:4]: I would avoid to be too much "negative" about performances.
I would remove the sentence after the comma; or maybe rephrase: "and performances not optimal".

File: docs/release-notes/s3gw-v0.1.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+**It is not intended for production use.**
```
Comment (giubacc) [inline docs/release-notes/s3gw-v0.1.0.md:6]: using instead something similar : "It is intended for testing and collecting feedback, production usage at this stage is not recommended" ? 

In this way we are clearing expressing our aim for this release, otherwise one could argue: "ok, so why should I install/try this?"

Comment (jecluis) [reply docs/release-notes/s3gw-v0.1.0.md:6]: good call

File: docs/release-notes/s3gw-v0.1.0.md (hunk 2 above)
Comment (jecluis) [reply docs/release-notes/s3gw-v0.1.0.md:4]: ack

File: docs/release-notes/s3gw-v0.1.0.md (hunk 1 above)
Comment (jecluis) [reply docs/release-notes/s3gw-v0.1.0.md:16]: meh, I think that kind of covers that. Let's give the recursive delete support as an easter egg for testers ;)

**Review by votdev — APPROVED** [review]
